pkgs/development/python-modules/thorlabspm100/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, unittestCheckHook
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  version = "1.2.2";
+  pname = "thorlabspm100";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "clade";
+    repo = "ThorlabsPM100";
+    rev = version;
+    sha256 = "WjX3IicycisncYBSBDOfDFd5RKv11suocTxoWw7WNHk=";
```

```suggestion
    hash = "sha256-WjX3IicycisncYBSBDOfDFd5RKv11suocTxoWw7WNHk=";
```
I think it's an SRI hash, but it's easy for others to understand like this.